src/Scouting.elm

```diff
@@ -0,0 +1,225 @@
+    , team : Maybe Int
+    , match : Maybe Int
+    , driverStation : String
+    , isStarted : ButtonState
+    }
+
+
+type ButtonState
+    = Pushed
+    | TriedPush
+    | Untouched
+
+
+init : Model
+init =
+    { scouter = ""
+    , team = Nothing
+    , match = Nothing
+    , driverStation = "    " -- for inputHelpMessage
+    , isStarted = Untouched
+    }
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        NameInput input ->
+            { model
+                | scouter = input
+            }
+
+        TeamInput input ->
+            { model
+                | team = String.toInt input
+                , driverStation = stationName (String.toInt input) model.match
+            }
+
+        MatchInput input ->
+            { model
+                | match = String.toInt input
+                , driverStation = stationName model.team (String.toInt input)
+            }
+
+        Start ->
+            { model
+                | isStarted = switchButtonState model
+                , driverStation = stationName model.team model.match
+            }
+
+
+ifCorrectInput : Model -> ButtonState
+ifCorrectInput model =
+    let
+        modelStr =
+            strModel model
+
+        isStation string =
+            model.driverStation == string
+    in
+    if List.any ((==) "" << (|>) modelStr) [ .scouter, .team, .match ] then
+        TriedPush
+
+    else if isStation " " || isStation "  " then
+        -- " " Not a match , "  " Team not in this match
+        TriedPush
+
+    else
+        Pushed
+
+
+switchButtonState : Model -> ButtonState
+switchButtonState model =
+    if model.isStarted == Pushed then
+        Untouched
+
+    else
+        ifCorrectInput model
+
+
+inputHelpMessage : String -> String
+inputHelpMessage strStation =
+    case strStation of
+        "    " ->
+            "Please fill all inputs^^"
+
+        "  " ->
+            "Team not in this match"
+
+        " " ->
+            "Not a match"
+
+        "" ->
+            "All inputs are required."
+
+        _ ->
+            ""
+
+
+startButton : StrModel -> Element.Element Msg
+startButton model =
+    let
+        ifStarted : String -> String -> String
+        ifStarted ifYes ifNo =
+            if model.isStarted == Pushed then
+                ifYes
+
+            else
+                ifNo
+    in
+    column [ Font.color (rgba 255 0 0 1) ]
+        [ button
+            [ Background.color (rgba 0 0 0 0.4)
+            , Font.color (rgba 1 1 1 1)
+            , Border.rounded 3
+            , width <| Element.px 50
+            ]
+            { onPress = Just Start
+            , label = text <| ifStarted "Startn't" "Start"
+            }
+        , text
+            << ifStarted ""
+          <|
+            inputHelpMessage model.driverStation
+        ]
+
+
+type alias StrModel =
+    { scouter : String
+    , team : String
+    , match : String
+    , driverStation : String
+    , isStarted : ButtonState
+    }

```

Comment:
Not sure why you need this StrModel... It might be convenient but I think it's more confusing.
I think the problem is with its name, as it is not really an all string types model and a second model in the same file is always confusing.
Maybe there is a better name.